lib/src/screens/home_screen.dart

```diff
@@ -1,10 +1,22 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import '../blocs/login_bloc/login_bloc.dart';
+import '../blocs/login_bloc/login_state.dart';
 
 class HomeScreen extends StatelessWidget {
   const HomeScreen({Key? key}) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
-    return Scaffold();
+    return BlocBuilder<LoginBloc, LoginState>(
+        bloc: BlocProvider.of<LoginBloc>(context),
+        builder: (context, state) {
+          String? userName = "no User";
+
+          print(userName);
```

Comment:
We should remove print statements for release.

Reply:
this screen is useless, i forgot to delete, sorry about that